app/controllers/admin/departments_controller.rb

```diff
@@ -17,7 +17,7 @@ class DepartmentsController < Admin::ApplicationController
     # for more information
 
     def find_resource(param)
-      Department.find_by! slug: param
+      Department.find param
```

Comment:
Yes sorry I may have hijacked this to echo the sentiment from @tobyhede as this is now happening in editorial now (not suggesting these changes did it).